### PR TITLE
JDK-8307381: Open Source JFrame, JIF related Swing Tests

### PR DESCRIPTION
**Pull request by honkar-jdk**
Open sourced the following JFrame, JIF related Swing tests.

```
test/jdk/javax/swing/JFrame/bug4101444.java
test/jdk/javax/swing/JFrame/bug4208018.java
test/jdk/javax/swing/JInternalFrame/bug4308938.java
test/jdk/javax/swing/JInternalFrame/bug4320889.java
test/jdk/javax/swing/JTextField/bug4300552.java
test/jdk/javax/swing/JToggleButton/bug4277049.java
```

CI testing looks good.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307381](https://bugs.openjdk.org/browse/JDK-8307381): Open Source JFrame, JIF related Swing Tests

### Reviewers
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Committer)
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13787/head:pull/13787` \
`$ git checkout pull/13787`

Update a local copy of the PR: \
`$ git checkout pull/13787` \
`$ git pull https://git.openjdk.org/jdk.git pull/13787/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13787`

View PR using the GUI difftool: \
`$ git pr show -t 13787`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13787.diff">https://git.openjdk.org/jdk/pull/13787.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13787#issuecomment-1533748565)

### CODE REVIEW

**Comment by mrserb** [comment]
Looks like the JBS issue is confidential: ⚠️ Temporary failure when trying to retrieve information on issue JDK-8307381

File: test/jdk/javax/swing/JToggleButton/bug4277049.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+
+/*
+ * @test
+ * @bug 4277049
+ * @summary Tests that ToggleButtonModel doesn't fire extra ChangeEvents
+ */
+
+public class bug4277049  implements ItemListener {
```
Comment (turbanoff) [inline test/jdk/javax/swing/JToggleButton/bug4277049.java:35]:
```suggestion
public class bug4277049 implements ItemListener {
```

**Comment by honkar-jdk** [comment]
> Looks like the JBS issue is confidential: ⚠️ Temporary failure when trying to retrieve information on issue JDK-8307381

@mrserb Looks to be resolved now.

**Review by DamonGuy — APPROVED** [review]

**Review by azuev-java — APPROVED** [review]
Looks good.

**Pull request closed without merging**